src/node/entities.h

```diff
@@ -58,7 +58,7 @@ namespace ccf
     static constexpr auto APP_SCRIPTS = "ccf.app_scripts";
     static constexpr auto SECRETS = "ccf.secrets";
     static constexpr auto CODE_IDS = "ccf.code_ids";
-    static constexpr auto VOTING_HISTORY = "ccf.voting_history";
```

Comment:
Having `ccf.governance_history` and `ccf.gov_scripts` seems odd and inconsistent. Can we have `ccf.governance.history` and `ccf.governance.scripts` instead?

Reply:
Done